less/components/profile.less

```diff
@@ -0,0 +1,33 @@
+@import "../variables";
+
+.profile-page {
+
+    p.success {
+        margin: @spacer/2;
+        color: #42a94b;
+        font-weight: 500;
+    }
+
+    p.failure {
+        margin: @spacer/2;
+        color: #a94442;
```

Comment:
If this & the success colour are likely to be re-used, let's variable-ify them and add to https://github.com/intermine/bluegenes/blob/dev/less/variables.less to prevent a hundred different similar failure/success definitions 😆